odoo/addons/base/models/res_lang.py

```diff
@@ -274,13 +274,16 @@ def create(self, vals_list):
 
     def write(self, vals):
         lang_codes = self.mapped('code')
+        inactive_users_lang = self.env['res.users'].search([('active', '=', False)]).mapped('lang')
```

Comment:
I don't see the connection with the spec.
Why don't you simply get OdooBot's lang with
```py
self.env.user.browse(SUPERUSER_ID).lang
```
And by the way, do that statement only when `active` is set to `False`, otherwise it costs an SQL query for nothing.